sdk/redis/arm-rediscache/test/redis_examples.ts

```diff
@@ -25,18 +25,6 @@ const replaceableVariables: Record<string, string> = {
 
 const recorderOptions: RecorderStartOptions = {
   envSetupForPlayback: replaceableVariables,
-  sanitizerOptions: {
```

Comment:
remove this rule as recorder will sanitize sensitive keys automatically

Reply:
could you share the recording link with me?

Reply:
https://github.com/Azure/azure-sdk-assets/blob/js/redis/arm-rediscache_195dc33567/js/sdk/redis/arm-rediscache/recordings/node/redis_test/recording_redis_create_test.json

search: primaryKey